tests/pytests/scenarios/compat/test_with_versions.py

```diff
@@ -53,13 +53,13 @@ def _get_test_versions():
             test_versions.append(
                 PySaltCombo(python_version=python_version, salt_version=salt_version)
             )
-    for salt_version in ("3001.4", "3002.2"):
+    for salt_version in ("3001.0", "3002.0", "3003.0", "3004.0"):
```

Comment:
3001 is out of support. Can we remove that version now?

Reply:
Of course!
How about the other even older than that?
Nuke those too?

Reply:
yep :)

Reply:
Fixed